framework/components/ecl-banners/ecl-banners.config.js

```diff
@@ -12,23 +12,9 @@ module.exports = {
       <!-- End: @${item.handle} -->\n
     `;
   },
-  default: 'background',
+  default: 'quote',
   variants: [
```

Comment:
We can even remove the variants, as there is only possibility left.
So config file could looks like this:
module.exports = {
  title: 'Banners',
  label: 'Banners',
  status: 'ready',
  tags: ['molecule'],
  context: {
    quote: {
      body: '
        Nullam eget gravida sapien. Curabitur nec ligula varius tellus blandit auctor in non felis.
        Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae;
        Pellentesque in volutpat eros, ut fringilla tortor.
      ',
      author: 'President Juncker',
    },
  },
};